spec/classes/archive/commmon_spec.rb

```diff
@@ -15,17 +33,11 @@
         git
         cmake
         gcc-c++
+        docker-compose-plugin
```

Comment:
I believe this spec change should be part of c169a06d24fb24dd891f57f45ddf7043a2494f8d

Reply:
moved